src/sentry/south_migrations/0432_auto__add_index_userreport_date_added__add_index_eventattachment_date_.py

```diff
@@ -0,0 +1,1231 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
I don't think we need this. These tables are pretty small and low volume, so the index will just take a second or so to create.

Reply:
👍